plugins/modules/system/shutdown.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
```

Comment:
```suggestion
# Copyright: (c) 2020, Ansible Project
```

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/700/commits/21bf25b0264a24e0cf5e2dc03f4c68a5b13d9401.

Reply:
Will mark this as resolved after double-checking with `sdoran` on IRC.